.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+### :bust_in_silhouette: Who
+_Who are you? If you are not Álvaro... Thanks and welcome! Could you explain a little bit who you are?_
+
+### :point_up: Goal(s)
+_Short description of the changes added_ 
+
+### Resolved issues:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-heading-punctuation] Don’t add a trailing `:` to headings](https://app.codacy.com/manual/alopezna5/mASAPP_CI/pullRequest?prid=5264864)